tests/system_tests/config.yaml

```
@@ -0,0 +1,13 @@
+api:
+  url: http:/0.0.0.0:8000
```

```suggestion
  url: http://0.0.0.0:8000
```

Interesting that this still works